libraries/tickfs/src/flash_controller.rs

```diff
@@ -0,0 +1,79 @@
+//! The Flash Controller interface with hardware
+
+use crate::error_codes::ErrorCode;
+
+/// Implementation required for the flash controller hardware. This
+/// should read, write and erase flash from the hardware using the
+/// flash controller.
+///
+/// This is the public trait for the Flash controller implementation.
+///
+/// The PageSize 'P' must be the smallest size that can be erased in
```

Comment:
Does `PageSize` refer to something in the code?

Reply:
Good catch, this is an out of date comment. I'll remove it.